### PR TITLE
Assignment5 alan and neriah

### PR DESCRIPTION
**Pull request by schemcharger**
By Alan Rose and Neriah Graham https://red-pebble-003ab421e.3.azurestaticapps.net/

### CODE REVIEW

**Comment by CKappes-Git** [comment]
I'm going to start working on a code review for this.

File: Wordle.Api/Wordle.Api/Controllers/WordController.cs [hunk 1]
```diff
@@ -29,18 +31,34 @@ public async Task<IEnumerable<Word>> GetManyWords(int? count)
             return await _wordService.GetSeveralWordsAsync(count);
         }
```
Comment (CKappes-Git) [inline Wordle.Api/Wordle.Api/Controllers/WordController.cs:33]: You'll need a service to get paginated words in the controller.

File: wordle-web/src/views/WordEditorView.vue [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+<script setup lang="ts">
+import Axios from 'axios'
+
+import { ListWord } from '@/scripts/listWord'
+import { ref } from 'vue'
+
+const motu = ref(true)
+
+const words = ref([
```
Comment (CKappes-Git) [inline wordle-web/src/views/WordEditorView.vue:68]: The word editor should be accessing words from the database.  It should do a paginated search for all of the words matching the search, and changes should be reflected in real time.

**Review by CKappes-Git — CHANGES_REQUESTED** [review]

**Review by meghanmae — COMMENTED** [review]
image

Looks like your menu option is missing?

Also, you had a few comments left by your peers regarding adding policies for your controllers.

**Pull request closed without merging**